packages/lib/gpc/tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "outDir": "dist",
     "declarationDir": "dist/types",
     // Include all source files, including tests
-    "rootDir": "."
+    "rootDir": ".",
+    "typeRoots": ["../gpcircuits/src/declarations"]
```

Comment:
What does this do?  Don't the explicit imports and exports cover bringing types into this package from gpcircuits?